stable/artifactory/templates/artifactory-statefulset.yaml

```diff
@@ -186,7 +186,7 @@ spec:
               key: master-key
         - name: EXTRA_JAVA_OPTIONS
           value: >
-      {{- with .Values.artifactory.javaOpts }}
+        {{- with .Values.artifactory.javaOpts }}
```

Comment:
Please fix this indentation :)

Reply:
This is aligned with what we have in `artifactory-ha`.

Reply:
👍